src/controller/disable.acount.controller.js

```diff
@@ -0,0 +1,45 @@
+import sendEmail from '../utils/sendEmail.js';
+import db from '../database/models/index.js';
+
+export const disableEnableUsers= async (req, res)=> {
+    const {status,reason,email}=req.body;
+
+    try{
+      const user = await db.User.findOne({ where: { email: email} });
+
+    if(!user){
+        return res.status(404).json({
+          message:`user with email : ${email} does not exit `
+      })
+    }
+
+      else{
+      user.status = status;
+
+      await user.save();
+
+      if (user) {
+        
+        const to = user.email;
+        const text=`Dear User, Your account has been ${status}  bacause of ${reason}.Please contact us if
```

Comment:
Since we know that it is the vendor that we are addressing, we start with "Dear Vendor". and also fix typo like "bacause"

Reply:
done